### ISSUE TITLE
Flag -m doesn't make Transmission-gtk start minimized

### ISSUE THREAD

**Issue by AlexNomadrg** [transmission/transmission#5094]
### What is the issue?

Transmission-gtk no longer starts minimized with minimized flag (probably after a recent update).

uname -a output `Linux arch-pc 6.1.12-arch1-1 #1 SMP PREEMPT_DYNAMIC Tue, 14 Feb 2023 22:08:08 +0000 x86_64 GNU/Linux`

![image](https://user-images.githubusercontent.com/58301160/221952858-1ded7d15-697e-46d2-8757-a932ade2d3fa.png)

### Which application of Transmission?

GTK+ app on Linux, BSD, etc.

### Which version of Transmission?

4.0.1 (e1c6e1be43)

**Comment by ckerr** [comment]
Possibly related to #4749?

**Comment by mikedld** [comment]
Nope, I'd rather say something's changed during C -> C++ transition, although I didn't test 3.00 yet to confirm.
